src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -188,40 +198,56 @@ export default {
                 image.selected = true;
             }
         },
-        stopListening() {
-            if (this.unsubscribe) {
-                this.unsubscribe();
-                delete this.unsubscribe;
+        boundsChange(bounds, isTick) {
+            this.bounds = bounds;
+            this.requestHistory(isTick);
+        },
+        requestHistory(isTick) {
+            if(!isTick) {
+                this.requestCount++;
+                const requestId = this.requestCount;
+                this.imageHistory = [];
+                this.openmct.telemetry
+                    .request(this.domainObject, this.bounds)
+                    .then((values = []) => {
+                        if (this.requestCount > requestId) {
+                            return Promise.resolve('Stale request');
```

Comment:
No need to return a promise here (suspect this is legacy code, but let's remove it.)

Reply:
@akhenry should I just return the string here or remove the check altogether?

Reply:
In the meantime, I changed it to just return the string. Nothing actually checks this so it might be moot either way.